cmd/faucet/fund.go

```diff
@@ -0,0 +1,106 @@
+package faucet
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+
+	cosmostypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+
+	"github.com/pokt-network/poktroll/cmd/flags"
+	"github.com/pokt-network/poktroll/cmd/logger"
+)
+
+// fundURLFmt is the canonical fund URL format for a given denom and recipient address.
+// The placeholders are intended to be interpolated in the following order:
+//   - baseURL: Fully-qualified URL to the faucet server (e.g. https://shannon-testnet-grove-faucet.beta.poktroll.com)
+//   - denom: the denom to fund (e.g. upokt)
+//   - recipientAddress: the recipient address to fund (e.g. pokt1mrqt5f7qh8uxs27cjm9t7v9e74a9vvdnq5jva4)
+const fundURLFmt = "%s/%s/%s"
+
+var faucetBaseURL string
+
+func FundCmd() *cobra.Command {
+	fundCmd := &cobra.Command{
+		Use:   "fund [denom] [recipient address]",
+		Args:  cobra.ExactArgs(2),
+		Short: "Request tokens of a given denom be sent to a recipient address.",
+		Long: `Request tokens of a given denom be sent to a recipient address.
+
+// TODO_UP_NEXT(@bryanchriswhite): update docs URL once known.
+For more information, see: https://dev.poktroll.com/operate/faucet`,
+		Example: `pocketd faucet fund upokt pokt1mrqt5f7qh8uxs27cjm9t7v9e74a9vvdnq5jva4
+pocketd faucet fund mact pokt1mrqt5f7qh8uxs27cjm9t7v9e74a9vvdnq5jva4 --base-url=http://localhost:8080`,
+		RunE: runFund,
+	}
+
+	fundCmd.Flags().StringVar(&faucetBaseURL, flags.FlagFaucetBaseURL, flags.DefaultFaucetBaseURL, flags.FlagFaucetBaseURLUsage)
```

Comment:
Should we mark this flag as required?

Reply:
I don't think so.
1. It has a reasonable default (Beta currently, w/ a comment to update to mainnet).
2. It can also set by `--network`.